src/api.rs

```diff
@@ -329,6 +331,7 @@ impl ChromaSampling {
   }
 }
 
+/// Contain the detail of the chroma sampling position ie. the location of chroma samples wrt luma samples.
```

Comment:
don't use `ie` or `wrt` when writing documentation

documentation needs to be readable, not short to write